Auto discover AA account if executen in AA

mortenlerudjordet opened the pull request:
Added logic for autodiscover of AA account, and made ResourceGroupName and AutomationAccountName optional input parameter. So the logic still can be run from outside AA.

AnatoliB commented:
@mortenlerudjordet Thank you, this PR looked like a good idea to me. I would just ask you to do it after [PR#6](https://github.com/microsoft/AzureAutomation-Account-Modules-Update/pull/6) is merged, to make sure Az modules are supported and not mixed with AzureRM.

mortenlerudjordet commented:
@AnatoliB : I modified Update-ModulesInAutomationToLatestVersion in the AA team github instead, and split it into two parts one for importing modules from PSGallery like Az, and one for updating either all modules or only the Azure ones. Liked the recursion of the module and did the work on that instead.

https://github.com/mortenlerudjordet/runbooks/blob/Dev/Utility/ARM/Import-PSGalleryModuleToAA.ps1

https://github.com/mortenlerudjordet/runbooks/blob/Dev/Utility/ARM/Update-PSGalleryModulesInAA.ps1
